### PR TITLE
Fixed scrolling of the main page in Safari.

### PR DESCRIPTION
**Pull request by siluk007**

### CODE REVIEW

File: src/components/Header.astro [hunk 1]
```diff
@@ -3,42 +3,58 @@ import Logo from './Logo.astro';
 import Menu from './Menu.astro';
 
 export interface Props {
-  logoColor?: "blue" | "white";
+  logoColor?: 'blue' | 'white';
+  backgroundColor?: 'bg-white' | 'bg-[#fde3d1]' | 'bg-transparent';
```
Comment (lemind) [inline src/components/Header.astro]: let's may be extract color `#fde3d1` to tailwind config

File: src/layouts/Layout.astro [hunk 2]
```diff
@@ -208,4 +208,43 @@ import '../index.css'
...
+  };
+
+  function updateScrollIndicator() {
+    const winScroll =
+      document.body.scrollTop || document.documentElement.scrollTop;
+    let height =
+      document.documentElement.scrollHeight -
+      document.documentElement.clientHeight;
+
+    var percent = getScrolledPercent(winScroll, height);
```
Comment (lemind) [inline src/layouts/Layout.astro]: `var`

**Review by Silenthunt0 — APPROVED** [review]

File: src/layouts/Layout.astro [hunk 3]
```diff
@@ -208,4 +208,43 @@ import '../index.css'
...
+    const scroll_indicator = document.getElementById('scroll-indicator');
```
Comment (lemind) [inline src/layouts/Layout.astro]: May we make all variables names as camelCase please

File: src/layouts/Layout.astro [hunk 4]
```diff
@@ -208,4 +208,43 @@ import '../index.css'
...
+    const envision = document.getElementById('envision');
+    const today = document.getElementById('today');
+    const header = document.getElementById('header');
+
+    if (envision && header) {
+      if (!envision.getBoundingClientRect().height) {
+				header.style.backgroundColor = "transparent"
+        return 0;
+      } else {
+				header.style.backgroundColor = "#fff";
```
Comment (lemind) [inline src/layouts/Layout.astro:241]: something with indentation

**Pull request merged**